packages-exp/auth-exp/src/core/auth/initialize.ts

```diff
@@ -44,6 +44,9 @@ export function _initializeAuthInstance(
     ? persistence
     : [persistence]
   ).map<Persistence>(_getInstance);
+  if (deps?.errorMap) {
+    auth._updateErrorMap(deps.errorMap);
```

Comment:
I didn't bring it up in the API review, but I wonder if setting the error map should be a global operation instead of a per-instance one because I think if people want to use the debug map, they want it to use it for all instances. No action needed now, we can wait and see if we got any feedback.

Reply:
Yeah I mentioned that yesterday in the biweekly check in as well. I definitely think that might be a good longer-term solution if it turns out people like this change